### DISCUSSION TITLE
Is it possible to specify the branch in a playbooks repo entry?

### DISCUSSION THREAD

**Discussion by jonah-loughlin-stfc** [semaphoreui/semaphore#759] [Q&A] [answered]
Basically just the title. I'd like to clone our dev branch rather than our master branch but I can't quite figure our how to do this.
Maybe this functionality isn't implemented yet?

Thanks in advance (:

**Comment by fiftin** [comment, marked as answer]
Hi @jTheCreator21,

https://docs.ansible-semaphore.com/user-guide/faq
